posthog/api/signup.py

```diff
@@ -15,16 +17,31 @@
 from social_django.strategy import DjangoStrategy
 
 from posthog.api.shared import UserBasicSerializer
+from posthog.cloud_utils import is_cloud
 from posthog.demo.matrix import MatrixManager
 from posthog.demo.products.hedgebox import HedgeboxMatrix
 from posthog.event_usage import alias_invite_id, report_user_joined_organization, report_user_signed_up
 from posthog.models import Organization, OrganizationDomain, OrganizationInvite, Team, User
 from posthog.permissions import CanCreateOrg
+from posthog.tasks.email import send_email_verification
 from posthog.utils import get_can_create_org
 
 logger = structlog.get_logger(__name__)
 
 
+def verify_email_or_login(request: HttpRequest, user: User) -> None:
+    require_verification_feature = posthoganalytics.feature_enabled("require-email-verification", str(user.uuid))
```

Comment:
@neilkakkar is this the correct way to check for a feature flag on the back-end?

Reply:
Do we have any speed requirements here?

Also, will the flag depend on any properties? Do you mind creating the flag in `app.posthog.com` with how it will be used?

Reply:
What do you mean by "speed requirements"? I'd like it to be relatively fast but I think the normal execution time would be fine?

The flag is simply for roll-out. Once we verify that everything works properly in production we can remove the flag.

Reply:
For posterity, @neilkakkar said in slack:

> Cool! Reason I ask is this will introduce a 300ms delay to every api call in production!
> Seems like it’s okay for now! So, looks good, full steam ahead :slightly_smiling_face: